tests/functional/pages/surveys/census/household/household-composition.page.js

```diff
@@ -0,0 +1,38 @@
+import QuestionPage from '../../question.page'
+
+class HouseholdCompositionPage extends QuestionPage {
```

Comment:
There is, it is slightly different. I thought we were going down the lines of a page model per page in survey.